x-pack/plugins/security_solution/public/sourcerer/experimental/components/dataview_picker/index.tsx

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { DataViewPicker } from '@kbn/unified-search-plugin/public';
+import React, { useCallback, useRef, useMemo, memo } from 'react';
+
+import { useKibana } from '../../../../common/lib/kibana/kibana_react';
+import { selectDataView } from '../../redux/actions';
+import { useDispatch, useSelector } from '../../redux/hooks';
+import { sourcererAdapterSelector } from '../../redux/selectors';
+
+const TRIGGER_CONFIG = {
+  label: 'Dataview',
+  color: 'danger',
+  title: 'Experimental data view picker',
+  iconType: 'beaker',
+} as const;
+
+export const DataviewPicker = memo(() => {
+  const dispatch = useDispatch();
+
+  const {
+    services: { dataViewEditor, data, dataViewFieldEditor },
+  } = useKibana();
+
+  const closeDataViewEditor = useRef<() => void | undefined>();
+  const closeFieldEditor = useRef<() => void | undefined>();
+
+  // TODO: should this be implemented like that?
+  // const canEditDataView =
+  // Boolean(dataViewEditor?.userPermissions.editDataView()) || !dataView.isPersisted();
```

Comment:
Yea, it looks like this is consistent with how it's done elsewhere:

```
  const canEditDataView =
    Boolean(dataViewEditor?.userPermissions.editDataView()) ||
    Boolean(selectedDataView && !selectedDataView.isPersisted());
 ```

Do you know how sourcerer accounted for permissions previously?